skins/laika/src/components/pages/donation_form/DateOfBirth.vue

```diff
@@ -0,0 +1,50 @@
+<template>
+    <fieldset>
+		<label for="birthDate" class="subtitle">{{ $t( 'membership_birth_date_label' ) }}</label>
```

Comment:
This label needs the class `has-margin-top-36` otherwise it's too close to the element above it.